src/test/java/subway/acceptance/line/SectionAcceptanceTest.java

```diff
@@ -34,9 +32,9 @@ void addLine() {
     @Test
     void createSection() {
         // when
-        var 이호선_요청 = LineRequestGenerator.이호선_요청_만들기(getStationId("강남역"), getStationId("역삼역"));
+        var 이호선_요청 = LineFixture.이호선_요청_만들기(getStationId("강남역"), getStationId("역삼역"));
```

Comment:
```suggestion
        var 이호선= LineFixture.이호선_요청_만들기(getStationId("강남역"), getStationId("역삼역"));
```

이호선으로 하는게 더 좋을 것 같네요! 이미 만들어졌으니까요 ㅎ